avro2py/codegen.py

```diff
@@ -338,9 +353,24 @@ def rewriter(self, module_namespace: str, module_imports: List[Union[ast.Import,
 class RewriteCrossReferenceStrings(ModuleAwareNodeTransformer):
     """Used to rewrite deferred type annotations within an AnnAssign node."""
 
+    def __init__(self, namespaces: Dict[str, Tuple[ast.AST, List[Union[ast.Import, ast.ImportFrom]]]]):
+        super(RewriteCrossReferenceStrings, self).__init__(namespaces)
+
+        namespace_parts = [namespace.split(".") for namespace in namespaces]
+        namespaces_with_children = [
+            ".".join(potential_parent)
+            for potential_parent, potential_child in itertools.permutations(namespace_parts, 2)
+            if namespace_is_parent_of(potential_parent, potential_child)
+        ]
+        self.namespaces_with_children = frozenset(namespaces_with_children)
```

Comment:
```suggestion
        self.namespaces_with_children = {
            ".".join(potential_parent)
            for potential_parent, potential_child in itertools.permutations(namespace_parts, 2)
            if namespace_is_parent_of(potential_parent, potential_child)
        }
```
so we just make the set directly from the get-go